config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -186,8 +187,13 @@ public void run() {
                     return;
                 }
                 if (configFuture.getOperation() == ConfigOperation.GET) {
-                    String result = CONFIG.getString(configFuture.getDataId());
-                    configFuture.setResult(result == null ? configFuture.getContent() : result);
+                    String result;
+                    try {
+                        result = CONFIG.getString(configFuture.getDataId());
+                    } catch (ConfigException.Missing missing){
```

Comment:
if file not exist，throw ConfigException.Missing?

Reply:
no, if file not exist, it will loading normally(loading the system properties)